### DISCUSSION TITLE
api 500 ?

### DISCUSSION THREAD

**Discussion by jerryc127** [walinejs/waline#1983] [Q&A] [answered]
最新评论 一直报 Input data should be a String

试着直接调用 GET /api/comment?type=count
返回 
```
{
    "errno": 500,
    "errmsg": "Input data should be a String"
}
```

后台也是，不能加载评论
![image](https://github.com/walinejs/waline/assets/16351105/10b0e6db-4550-49b7-b570-b4dba58b9952)

---

api 我之前调用正常
但是有一次调用   /api/comment?type=count 的时候，我用 post 去调用，之后就不行了，返回 500

---
Latest comments have been reported Input data should be a String

Try calling GET /api/comment?type=count directly
return 
```
{
    "errno": 500,
    "errmsg": "Input data should be a String"
}
```

Also in the background, comments cannot be loaded
![image](https://github.com/walinejs/waline/assets/16351105/10b0e6db-4550-49b7-b570-b4dba58b9952)

---

The api I called before was normal
But when I called /api/comment?type=count once, I used post to call it, and then it didn’t work, and returned 500

**Comment by lizheming** [comment]
我在官网试了下无法复现这个问题。提供下你的地址和部署方式吧，有问题可以选择 QA 分类，提供好详细的信息方便排查。

---
I tried on the official website and couldn't reproduce this problem. Provide your address and deployment method. If you have any questions, you can choose QA classification and provide detailed information for easy troubleshooting.

**Comment by jerryc127** [comment]
https://butterfly.js.org/
https://btf-waline.crazywong.com/


部署是 vercel + MongoDB

后台能加载 垃圾这个分类 已通过和待审核不能加载

**Comment by jerryc127** [comment, marked as answer]
我在 MongoDB 发现这一条
估计是我 调用api写入的
删了之后就正常了

```
{
  "_id": {
    "$oid": "6469c155e1934a000fd5accc"
  },
  "link": null,
  "mail": null,
  "nick": null,
  "pid": null,
  "rid": null,
  "ua": null,
  "url": null,
  "comment": null,
  "ip": "隐私删除",
  "insertedAt": {
    "$date": {
      "$numberLong": "1684652372558"
    }
  },
  "user_id": null,
  "status": "approved"
}
```
